server/app/views/admin/programs/ProgramFormBuilder.java

```diff
@@ -328,7 +328,10 @@ private DivTag showCategoryCheckboxes(
                                             .withType("checkbox")
                                             .withName("categories" + Path.ARRAY_SUFFIX)
                                             .withValue(String.valueOf(category.getId()))
-                                            .withCondChecked(categories.contains(category.getId())),
+                                            .withCondDisabled(isCommonIntakeForm)
+                                            .withCondChecked(
+                                                categories.contains(category.getId())
```

Comment:
Non-blocking- Is there a reason to add it twice- both in withCondDisable and withCondChecked

Reply:
Since `disabled` and `checked` are two different HTML attributes, I don't think they can share the same conditional statement.  If you're picturing a way to do that, would you mind giving an example?

Reply:
This question is more for my understanding.